pkg/config/reader.go

```diff
@@ -99,7 +165,21 @@ func (r *FileReader) parseConfig() error {
 
 		// Needed for forbidigo.
 		mapstructure.TextUnmarshallerHookFunc(),
-	))); err != nil {
+	)), func(config *mapstructure.DecoderConfig) {
+		config.ErrorUnused = true
+		if os.Getenv("HELP_RUN") == "2" {
```

Comment:
I had cyclic imports errors here. Maybe add all envVars names to a single file? related to https://github.com/golangci/golangci-lint/pull/3196